### PR TITLE
VTC-5523 add BST connector and feature switch

### PR DESCRIPTION
**Pull request by MarkAKelly**

### CODE REVIEW

File: app/businessrates/authorisation/connectors/BackendConnector.scala [hunk 1]
```diff
@@ -16,50 +16,19 @@
...
-import play.api.libs.json.Reads
-import uk.gov.hmrc.http.HttpReads._
-import uk.gov.hmrc.http.{HeaderCarrier, NotFoundException}
-import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import businessrates.authorisation.models.{Organisation, Person}
+import uk.gov.hmrc.http.HeaderCarrier
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class BackendConnector @Inject()(val http: VOABackendWSHttp, servicesConfig: ServicesConfig)
```
Comment (PeterPerhac) [inline app/businessrates/authorisation/connectors/BackendConnector.scala:28]: nice cleanup here, thanks

File: app/businessrates/authorisation/config/FeatureSwitch.scala [hunk 2]
```diff
@@ -14,13 +14,18 @@
...
 
-import scala.concurrent.{ExecutionContext, Future}
-import uk.gov.hmrc.http.HeaderCarrier
+import javax.inject.Inject
 
-trait PersonAccounts {
-  def getPerson(externalId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[Person]]
+class FeatureSwitch @Inject()(config: Configuration) {
+
+  private val prefix = "feature-switch"
```
Comment (PeterPerhac) [inline app/businessrates/authorisation/config/FeatureSwitch.scala:25]: while there's nothing wrong with "feature-switch" prefix, I would prefer just "feature"
it's configuring/enabling a specific feature. The "switch" is the mechanism that uses this config to do the switching.

Comment (PeterPerhac) [reply app/businessrates/authorisation/config/FeatureSwitch.scala:25]: I know, you have already raised 10 other PRs with config changes 😬 so I guess you will not want to change it now. But I would still like that

**Comment by PeterPerhac** [comment]
👍 looks very good to me

**Comment by habeeb01** [comment]
LGTM 👍

**Pull request merged**